dev/bazel/HOWTO.md

```diff
@@ -292,9 +341,21 @@ dal_test_suite(
 ```
 
 ### Run tests for the existing oneDAL build
-TBD
+1. Set `DAALROOT` env var:
```

Comment:
At the moment, we have `DALROOT` variable - I think it is better to use this one. `DAALROOT` is just for compatibility I guess

Reply:
In fact Bazel uses DAALROOT now, need to fix in Bazel scripts and then in docs